### DISCUSSION TITLE
Launch Federated Learning with multiple clients

### DISCUSSION THREAD

**Discussion by bthassen** [Project-MONAI/MONAI#3270] [General]
Hi, i'am currently trying to figure out how to launch the example of FL from the tutorials, using multiple computers as clients and my own computer to run the server and the admin containers. I'am struggeling to that due to my  poor Docker container Knowledge. Is there someone here who already done this. 
Ps: it already worked on a single computer

**Comment by Nic-Ma** [comment]
Hi @bthassen ,

Thanks for your interest here.
Could you please help share the tutorial link you are talking about?
Add @yiheng-wang-nv to double-check further.

Thanks in advance.

**Reply by bthassen** [reply]
hi thanks for the quick reply. I mean this tutorial: https://github.com/Project-MONAI/tutorials/tree/master/federated_learning/nvflare/nvflare_example_docker.
@yiheng-wang-nv
